docs/pages/setup/reference/cli.mdx

```diff
@@ -743,6 +743,31 @@ $ tctl users add joe joe,root
 $ tctl users add joe
 ```
 
+### tctl users update
+
+Update user account:
+
+```code
+$ tctl users update [<flags>] <account>
+```
+
+#### Arguments
+
+- `<account>` - The Teleport user account name.
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--set-roles` | none | a role name | Comma-separated list of roles for the user to assume |
```

Comment:
Does this add to the user's roles or replace them? Would be good to clarify.

Reply:
Good call, documented (it replaces the roles).